examples/anonymous_auth.rb

```diff
@@ -27,4 +27,4 @@ def run_authentication(address, smb1, smb2, smb3, username, password)
 password = ''
 
 # Negotiate with only SMB1 enabled
-run_authentication(address, true, false, false, username, password)
+run_authentication(address, true, true, true, username, password)
```

Comment:
Would it be possible to also update this example script with the same argument parsing logic as in `examples/tree_connect.rb` for the SMB versions? This would enable the user to quickly test multiple SMB version combinations without having to update the code.

Reply:
👍🏻 For sure, I'd be happy to. In the future I think all of the examples should have argument parsing, it just makes things easier to test and tweak.